js/background/background.js

```diff
@@ -736,6 +736,76 @@ class Steam {
 Steam._dynamicstore_promise = null;
 Steam._supportedCurrencies = null;
 
+// When redirecting to Steam Community, if the game does not actually exist, onHeadersReceived would normally trigger an infinite redirection loop. To avoid that, we store the IDs of the requests that have already been redirected here.
+let webRequests = [];
+
+function onHeadersReceived(details) {
+    let response = {};
+
+    if (details.statusCode === 302 && webRequests.indexOf(details.requestId) < 0) {
+        let matches = details.url.match(/(app|sub)\/(\d+)/);
+
+        if (matches) {
+            let type = matches[1];
+            let id = matches[2];
+
+            switch (SyncedStorage.get('redirect_removed_to')) {
+                case 'community':
+                    response.redirectUrl = `https://steamcommunity.com/${type}/${id}`;
```

Comment:
afaik subs don't have a community page.
Maybe redirect to SteamDB then?
What do you think?

Reply:
Ok, should I add an option to allow choosing the redirect for apps / subs separately?